### ISSUE TITLE
[BUG] Data Model Attributes

### ISSUE THREAD

**Issue by tommartensen** [bptlab/gryphon#30]
I had to rename the attribute of a Data class (because Chimera rejected it; probably because of an underscore). This was reflected in the UI. 
In the Scenario Export however, there were two attributes for this data class: 
1. The old one
1. The renamed one

This of course fails the deployment to Chimera.

**Comment by marhew** [comment]
Did Gryphon display a warning like "only alpha-numeric names allowed?" when you saved the data model? I will try to re-create this bug.

**Comment by tommartensen** [comment]
I don't think so. To mitigate, we could introduce a regex matcher for the attribute names.

**Comment by marhew** [comment]
https://github.com/bptlab/gryphon/blob/gh-pages/_includes/Howto.md#naming-conventions

**Comment by tommartensen** [comment]
:+1: So is that same regex used in Chimera? If not, these naming conventions should be harmonized.
